internal/plugins/ansible/v1/scaffolds/api.go

```diff
@@ -88,8 +88,8 @@ func (s *apiScaffolder) scaffold() error {
 
 	// Check that the provided group can be added to the project
 	if !s.config.MultiGroup && len(s.config.Resources) != 0 && !s.config.HasGroup(resourceOptions.Group) {
-		return fmt.Errorf("multiple groups are not allowed by default, to enable multi-group visit %s",
-			"kubebuilder.io/migration/multi-group.html")
+		return fmt.Errorf("multiple groups are not allowed by default, %s",
+			"to enable multi-group add multigroup: true into the PROJECT file.")
```

Comment:
Since the entire string is static, no need to use `%s`. Use `errors.New()` instead. Also, put backticks around `multigroup: true` to make the instruction more obvious.

```suggestion
		return errors.New("support for multiple groups is disabled; to enable, set `multigroup: true` in the PROJECT file.")
```